components/discord_bot/actions/remove-user-role/remove-user-role.mjs

```diff
@@ -0,0 +1,58 @@
+import common from "../common.mjs";
```

Comment:
Since this component doesn't use `channelId`, you can include common.mjs from the discord_bot directory.
`import common from "../../common.mjs"`;